diff -Nur exim-texinfo-4.60.orig/doc/filter.texinfo exim-texinfo-4.60/doc/filter.texinfo
--- exim-texinfo-4.60.orig/doc/filter.texinfo	Wed Nov 30 18:46:57 2005
+++ exim-texinfo-4.60/doc/filter.texinfo	Wed Nov 30 18:50:13 2005
@@ -2,10 +2,15 @@
 @setfilename exim_filtering.info
 @documentencoding us-ascii
 @settitle Exim's interfaces to mail filtering
+@ifinfo
+@format
+@dircategory Net utilities:
 @direntry
 * Exim's interfaces to mail filtering: (exim_filtering).
                                                   [MISSING TEXT]
 @end direntry
+@end format
+@end ifinfo
 
 @node Top, Forwarding and filtering in Exim, , (dir)
 @documentlanguage en
diff -Nur exim-texinfo-4.60.orig/doc/spec.texinfo exim-texinfo-4.60/doc/spec.texinfo
--- exim-texinfo-4.60.orig/doc/spec.texinfo	Wed Nov 30 18:46:57 2005
+++ exim-texinfo-4.60/doc/spec.texinfo	Wed Nov 30 18:52:03 2005
@@ -1,11 +1,16 @@
 \input texinfo
-@setfilename the_exim_mta.info
+@setfilename exim.info
 @documentencoding us-ascii
 @settitle Specification of the Exim Mail Transfer Agent
+@ifinfo
+@format
+@dircategory Net utilities:
 @direntry
 * Specification of the Exim Mail Transfer Agent: (the_exim_mta).
                                                   [MISSING TEXT]
 @end direntry
+@end format
+@end ifinfo
 
 @node Top, Introduction, , (dir)
 @documentlanguage en
