diff -urN exim-3.15.orig/exim_monitor/EDITME exim-3.15/exim_monitor/EDITME
--- exim-3.15.orig/exim_monitor/EDITME	Wed Jun 14 09:57:45 2000
+++ exim-3.15/exim_monitor/EDITME	Sat Jul 15 18:45:44 2000
@@ -149,7 +149,7 @@
 # will be displayed as the second stripchart. This can be used to keep
 # a display of a mail spool partition on the screen.
 
-# SIZE_STRIPCHART=/var/mail
+SIZE_STRIPCHART=/var/spool/exim
 
 # The name of the size stripchar will be the last component of SIZE_STRIPCHART
 # unless the following variable is set to override it.
