diff -urN exim-4.64.org/OS/Makefile-Default exim-4.64/OS/Makefile-Default
--- exim-4.64.org/OS/Makefile-Default	2006-12-20 10:46:04.000000000 +0100
+++ exim-4.64/OS/Makefile-Default	2006-12-21 16:39:39.098652250 +0100
@@ -27,7 +27,7 @@
 # found, it uses /usr/ucb/basename. This copes with Solaris 2 and Linux, both
 # of which come in different versions.
 
-BASENAME_COMMAND=/usr/bin/basename
+BASENAME_COMMAND=/bin/basename
 
 
 # If you set STRIP_COMMAND to the path of the "strip" command, it will be run
@@ -40,8 +40,8 @@
 # Some of the following commands live in different places in different OS. We
 # include them all here for generality.
 
-CHOWN_COMMAND=/usr/bin/chown
-CHGRP_COMMAND=/usr/bin/chgrp
+CHOWN_COMMAND=/bin/chown
+CHGRP_COMMAND=/bin/chgrp
 MV_COMMAND=/bin/mv
 RM_COMMAND=/bin/rm
 
@@ -175,7 +175,7 @@
 # parameters for include and library directories that may be needed for IPv6 on
 # some systems, where the support is not yet in the standard library.
 
-# HAVE_IPV6=YES
+HAVE_IPV6=YES
 # IPV6_INCLUDE=-I /usr/ipv6/include
 # IPV6_LIBS=-L/usr/ipv6/libs -linet6
 

