diff -urN exim-texinfo-4.00.org/doc/filter.texinfo exim-texinfo-4.00/doc/filter.texinfo
--- exim-texinfo-4.00.org/doc/filter.texinfo	Sat Jun 30 01:12:24 2001
+++ exim-texinfo-4.00/doc/filter.texinfo	Sat Jun 30 01:12:53 2001
@@ -4,10 +4,19 @@
 @set{wmAuthor_email} <ph10@@cus.cam.ac.uk>
 @set{COPYRIGHT1} Copyright @copyright{} @value{wmYear} University of Cambridge
 @c %**start of header
-@setfilename filter.info
+@setfilename exim_filter.info
 @settitle Exim Filter Specification
 @paragraphindent 0
 @c %**end of header
+
+@ifinfo
+@format
+@dircategory Net utilities:
+@direntry
+* Exim Filters: (exim_filter).		Message Transfer Agent
+@end direntry
+@end format
+@end ifinfo
 
 @titlepage
 @title The Exim Mail Transfer Agent
diff -urN exim-texinfo-4.00.org/doc/spec.texinfo exim-texinfo-4.00/doc/spec.texinfo
--- exim-texinfo-4.00.org/doc/spec.texinfo	Sat Jun 30 01:12:24 2001
+++ exim-texinfo-4.00/doc/spec.texinfo	Sat Jun 30 01:12:54 2001
@@ -4,10 +4,19 @@
 @set{wmAuthor_email} <ph10@@cus.cam.ac.uk>
 @set{COPYRIGHT1} Copyright @copyright{} @value{wmYear} University of Cambridge
 @c %**start of header
-@setfilename spec.info
+@setfilename exim.info
 @settitle Exim Specification
 @paragraphindent 0
 @c %**end of header
+
+@ifinfo
+@format
+@dircategory Net utilities:
+@direntry
+* Exim: (exim).				Message Transfer Agent
+@end direntry
+@end format
+@end ifinfo
 
 @titlepage
 @title The Exim Mail Transfer Agent
