diff -urN exim-4.22.org/src/EDITME exim-4.22/src/EDITME
--- exim-4.22.org/src/EDITME	2003-09-03 23:36:28.000000000 +0200
+++ exim-4.22/src/EDITME	2003-09-03 23:38:08.000000000 +0200
@@ -98,7 +98,7 @@
 # /usr/local/sbin. The installation script will try to create this directory,
 # and any superior directories, if they do not exist.
 
-BIN_DIRECTORY=/usr/exim/bin
+BIN_DIRECTORY=/usr/bin
 
 
 #------------------------------------------------------------------------------
@@ -114,7 +114,7 @@
 # don't exist. It will also install a default runtime configuration if this
 # file does not exist.
 
-CONFIGURE_FILE=/usr/exim/configure
+CONFIGURE_FILE=/etc/mail/exim.conf
 
 # It is possible to specify a colon-separated list of files for CONFIGURE_FILE.
 # In this case, Exim will use the first of them that exists when it is run.
@@ -132,7 +132,8 @@
 # owner of a local mailbox.) Specifying these values as root is very strongly
 # discouraged. These values are compiled into the binary.
 
-EXIM_USER=
+EXIM_UID=79
+EXIM_GID=79
 
 # If the setting of EXIM_USER is numeric (e.g. EXIM_USER=42), there must
 # also be a setting of EXIM_GROUP. If, on the other hand, you use a name
@@ -213,7 +214,7 @@
 # This one is special-purpose, and commonly not required, so it is not
 # included by default.
 
-# TRANSPORT_LMTP=yes
+TRANSPORT_LMTP=yes
 
 
 #------------------------------------------------------------------------------
@@ -222,9 +223,9 @@
 # MBX, is included only when requested. If you do not know what this is about,
 # leave these settings commented out.
 
-# SUPPORT_MAILDIR=yes
-# SUPPORT_MAILSTORE=yes
-# SUPPORT_MBX=yes
+SUPPORT_MAILDIR=yes
+SUPPORT_MAILSTORE=yes
+SUPPORT_MBX=yes
 
 
 #------------------------------------------------------------------------------
@@ -239,18 +240,18 @@
 LOOKUP_DBM=yes
 LOOKUP_LSEARCH=yes
 
-# LOOKUP_CDB=yes
-# LOOKUP_DNSDB=yes
-# LOOKUP_DSEARCH=yes
+LOOKUP_CDB=yes
+LOOKUP_DNSDB=yes
+LOOKUP_DSEARCH=yes
 # LOOKUP_LDAP=yes
 # LOOKUP_MYSQL=yes
 # LOOKUP_NIS=yes
 # LOOKUP_NISPLUS=yes
 # LOOKUP_ORACLE=yes
-# LOOKUP_PASSWD=yes
+LOOKUP_PASSWD=yes
 # LOOKUP_PGSQL=yes
 # LOOKUP_WHOSON=yes
-# LOOKUP_WILDLSEARCH=yes
+LOOKUP_WILDLSEARCH=yes
 
 
 #------------------------------------------------------------------------------
@@ -310,9 +311,9 @@
 # included in the Exim binary. You will then need to set up the run time
 # configuration to make use of the mechanism(s) selected.
 
-# AUTH_CRAM_MD5=yes
-# AUTH_PLAINTEXT=yes
-# AUTH_SPA=yes
+AUTH_CRAM_MD5=yes
+AUTH_PLAINTEXT=yes
+AUTH_SPA=yes
 
 
 #------------------------------------------------------------------------------
@@ -390,10 +391,10 @@
 # leave these settings commented out.
 
 # This setting is required for any TLS support (either OpenSSL or GnuTLS)
-# SUPPORT_TLS=yes
+SUPPORT_TLS=yes
 
 # Uncomment this setting if you are using OpenSSL
-# TLS_LIBS=-lssl -lcrypto
+TLS_LIBS=-lssl -lcrypto
 
 # Uncomment these settings if you are using GnuTLS
 # USE_GNUTLS=yes
@@ -444,7 +445,7 @@
 # Once you have done this, "make install" will build the info files and
 # install them in the directory you have defined.
 
-# INFO_DIRECTORY=/usr/share/info
+INFO_DIRECTORY=/usr/share/info
 
 
 #------------------------------------------------------------------------------
@@ -457,7 +458,7 @@
 # %s. This will be replaced by one of the strings "main", "panic", or "reject"
 # to form the final file names. Some installations may want something like this:
 
-# LOG_FILE_PATH=/var/log/exim_%slog
+LOG_FILE_PATH=/var/log/exim/%s.log
 
 # which results in files with names /var/log/exim_mainlog, etc. The directory
 # in which the log files are placed must exist; Exim does not try to create
@@ -506,7 +507,7 @@
 # files. Both the name of the command and the suffix that it adds to files
 # need to be defined here. See also the EXICYCLOG_MAX configuration.
 
-COMPRESS_COMMAND=/usr/bin/gzip
+COMPRESS_COMMAND=/bin/gzip
 COMPRESS_SUFFIX=gz
 
 
@@ -514,7 +515,7 @@
 # If the exigrep utility is fed compressed log files, it tries to uncompress
 # them using this command.
 
-ZCAT_COMMAND=/usr/bin/zcat
+ZCAT_COMMAND=/bin/zcat
 
 
 #------------------------------------------------------------------------------
@@ -523,7 +524,7 @@
 # (version 5.004 or later) installed, set EXIM_PERL to perl.o. Using embedded
 # Perl costs quite a lot of resources. Only do this if you really need it.
 
-# EXIM_PERL=perl.o
+EXIM_PERL=perl.o
 
 
 #------------------------------------------------------------------------------
@@ -533,11 +534,12 @@
 # support, which is intended for use in conjunction with the SMTP AUTH
 # facilities, is included only when requested by the following setting:
 
-# SUPPORT_PAM=yes
+SUPPORT_PAM=yes
 
 # You probably need to add -lpam to EXTRALIBS, and in some releases of
 # GNU/Linux -ldl is also needed.
 
+EXTRALIBS=-lpam -ldl
 
 #------------------------------------------------------------------------------
 # Support for authentication via Radius is also available. The Exim support,
@@ -562,7 +564,7 @@
 # /var/pwcheck) and chown it to exim's user and group. Once you have installed
 # pwcheck, you should arrange for it to be started by root at boot time.
 
-# CYRUS_PWCHECK_SOCKET=/var/pwcheck/pwcheck
+CYRUS_PWCHECK_SOCKET=/var/lib/sasl/mux
 
 
 #------------------------------------------------------------------------------
@@ -579,7 +581,7 @@
 # group. Once you have installed saslauthd, you should arrange for it to be
 # started by root at boot time.
 
-# CYRUS_SASLAUTHD_SOCKET=/var/state/saslauthd/mux
+CYRUS_SASLAUTHD_SOCKET=/var/lib/sasl2/mux
 
 
 #------------------------------------------------------------------------------
@@ -691,7 +693,7 @@
 # period (for example, /usr/exim/configure.host.in.some.domain). If this file
 # does not exist, then the bare configuration file name is tried.
 
-# CONFIGURE_FILE_USE_NODE=yes
+CONFIGURE_FILE_USE_NODE=yes
 
 
 #------------------------------------------------------------------------------
@@ -841,7 +843,7 @@
 # (process id) to a file so that it can easily be identified. The path of the
 # file can be specified here. Some installations may want something like this:
 
-# PID_FILE_PATH=/var/lock/exim.pid
+PID_FILE_PATH=/var/run/exim%s.pid
 
 # If PID_FILE_PATH is not defined, Exim writes a file in its spool directory
 # using the name "exim-daemon.pid".
@@ -864,7 +866,7 @@
 # be changed here. The default is 0640 so that information from the spool is
 # available to anyone who is a member of the Exim group.
 
-# SPOOL_MODE=0640
+SPOOL_MODE=0640
 
 
 #------------------------------------------------------------------------------
